testing/traefik/APKBUILD

```diff
@@ -24,24 +25,27 @@ builddir="$srcdir/src/github.com/containous/$pkgname"
 #   - CVE-2019-9512
 #   - CVE-2019-9514
 
+export GOPATH="$srcdir"
+
 prepare() {
 	default_prepare
 	mkdir -p "$srcdir/src/github.com/containous"
 	mv "$srcdir"/$pkgname-$pkgver "$builddir"/
+#	CGO_ENABLED=0 GO111MODULE=off go get github.com/containous/go-bindata/...
```

Comment:
It should be there according to https://docs.traefik.io/contributing/building-testing/#build-traefik

But it builds and works fine without this bindata